src/entry/mod.rs

```diff
@@ -0,0 +1,15 @@
+use hopper;
+use metric::Event;
+
+pub trait Entry {
+    // fn run();
+    fn get_config(&self) -> &EntryConfig;
+    fn run1(&mut self, _forwards: Vec<hopper::Sender<Event>>, recv: hopper::Receiver<Event>);
```

Comment:
Well, this is the reason why I made a PR so early.
I don't know how to structure the code so that I didn't have this function.
I mean, the only thing that `run1` does - it calls `run`.
But as the Trait is defined for different types (Sink, Source, Filter) - I can't write a proper trait that will ensure that the function exists.
To rephrase: 
I need a new trait (`Entry` in my code) that have function (`run1`), such that this function would start all types of objects (`Sink`, `Source`, `Filter`).
We already have `run` for `Sink`, `Source`, `Filter`, so I can't make `Entry` have `run`. This is why I defined `run1`, and now I have to write implementation for  it in all sinks, in all sources and filters. Though, it has the same implementation `fn run1() -> run() end` (in pseudocode).
I hope that I missed something in language syntax so that I have trait `Entry` with default `run1` function. Is it possible?